eng/pipelines/templates/jobs/test-matrix.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
```

Comment:
Should we call this "job-matrix" instead so that we don't have folks think this is only for testing (although that is its primary usage).

Reply:
Yeah. `test-matrix` felt a little misleading to me as well but I couldn't think of a better name at the time.